.gitignore

```diff
@@ -42,3 +49,12 @@ luac.out
 *.x86_64
 *.hex
 
+### VisualStudioCode ###
```

Comment:
I think this is the wrong approach, local files from the IDE each person is using should never be included in the project's `. gitignore`, [but on the global one](https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/). The same is true for `.DS_Store` and the like. Can you please remove all new lines to this file?

Reply:
okay sounds fair. Thanks for the reference.